dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobConfiguration.java

```diff
@@ -87,6 +87,8 @@
     private boolean inMemoryJob = false;
 
     private String userUid;
+    
+    private boolean leaderOnlyJob = false;
```

Comment:
false here, but true in the .hbm file? What is the correct default value?

Reply:
The general idea is to have leaderOnlyJob flag set to true for all jobs, which is why I set it to true in hbm. But after few rounds of testing, figured out that for the inMemoryJobs that are scheduled or created at runtime, there is always a new JobConfiguration object that is created. For such jobConfiguration it would make sense to default the flag to false, since the job has to be run by the instance in which the inMemoryJob is created irrespective of whether the instance is leader or not. Maybe I can change the default in hbm also to false. Since we have the TableAlteror setting all values to true anyway. Sounds ok?

Reply:
I think its a good convention and more natural to leave booleans with default = false. If leader only is the norm, could we reframe and call the property `parallelJob` ?